README.md

```diff
@@ -166,3 +166,4 @@ The example application is divided into four services which perform actions on a
 
 ## License
 This project is licensed under [Eclipse Public License - v 2.0](LICENSE). 
+
```

Comment:
This line was added/removed/added to prompt a re-build to test the build process. I can remove it again to put things back to how they were.